scipy/linalg/tests/test_basic.py

```diff
@@ -154,12 +154,18 @@ def test_bad_shape(self):
         assert_raises(ValueError, solve_banded, (1, 1), ab, [1.0, 2.0])
 
     def test_1x1(self):
+        # nupper == 1, nlower == 1
         b = array([[1., 2., 3.]])
         x = solve_banded((1, 1), [[0], [2], [0]], b)
         assert_array_equal(x, [[0.5, 1.0, 1.5]])
         assert_equal(x.dtype, np.dtype('f8'))
         assert_array_equal(b, [[1.0, 2.0, 3.0]])
 
+        # nupper == nlower == 0
+        # NOTE: 3/2 is representable exactly in binary floating,
+        # so we use strict equality test instead of allmost_equal
```

Comment:
minor comment -- typo in `allmost_equal`; could switch to mentioning the recommended-for-floats `assert_allclose` which you apparently don't need here due to exactness